lib/Basic/CMakeLists.txt

```diff
@@ -15,3 +15,8 @@ add_llbuild_library(llbuildBasic
 if(${CMAKE_SYSTEM_NAME} MATCHES ".*BSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   target_link_libraries(llbuildBasic pthread)
 endif()
+
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+  link_directories(${LLBUILD_WINDOWS_UM_DIR})
```

Comment:
Why is CMake not able to figure this out on its own? That directory should already be in the search path when building from a VS command prompt AFAIK.